app/models/user_content_report.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class UserContentReport
+  def initialize(reported_content:)
+    unless [Comment, RequestForComment].include?(reported_content.class)
+      raise("#{reported_content.model_name} is not configured for spam reports.")
```

Comment:
this could also be adjusted to reflect the new class name

Reply:
I am sorry; I am not sure who I can reflect the the new class name here.

Reply:
see other comment